tests/test_visitors/test_ast/test_functions/test_getter_without_return.py

```diff
@@ -0,0 +1,249 @@
+import pytest
+
+from wemake_python_styleguide.violations.best_practices import (
+    GetterWithoutReturnViolation,
+)
+from wemake_python_styleguide.visitors.ast.functions import (
+    FunctionSignatureVisitor,
+)
+
+getter_function_with_implicit_return = """
+def get_foo():
+    ...
+"""
+
+getter_function_with_bare_return = """
+def get_foo():
+    return
+"""
+
+getter_function_with_valued_return = """
+def get_foo():
+    return 1
+"""
+
+getter_function_with_explicit_none_return = """
+def get_foo():
+    return None
+"""
+
+getter_function_with_bare_yield = """
+def get_foo():
+    yield
+"""
+
+getter_function_with_valued_yield = """
+def get_foo():
+    yield 1
+"""
+
+getter_function_with_explicit_none_yield = """
+def get_foo():
+    yield None
+"""
+
+getter_function_with_yield_from = """
+def get_foo():
+    yield from [1]
+"""
+
+getter_method_with_implicit_return = """
+class Foo:
+    def get_foo():
```

Comment:
NIP: please, let's add `self` argument to all methods - it doesn't change anything, but method without `self` argument or `staticmethod` decorator is confusing

Reply:
Good catch 👍